### ISSUE TITLE
Gh action deploy without --prod

### ISSUE THREAD

**Issue by nicrosengren** [denoland/deployctl#227]
I've tried to find a way to allow gh actions to only do preview deploys.
It seems to perform deploys in a similar way to when the --prod flag i sent to the cli. 

If the default must be --prod it would be really nice to have a config flag that would only do preview deploys that requires manual promotion.

Sorry if this exists and Ive missed it, in that case I would gladly submit a PR adding it to the action README.

**Comment by arnauorriols** [comment]
When you link a Deploy project to a Github repository in order to use the GH action, you set which branch will act as the "production" branch. Any commit in that branch will result in a production deployment. I presume you have only 1 branch (main), in which case you could create a new branch and set it as the "production" branch. You can forget about it if you want to promote deployments manually, or you could merge to it those commits that should be deployed to production.

**Comment by nicrosengren** [comment]
Thanks for your reply!
We've used a separate production branch before but what we're running now is a tag based versioning system and branching from tag on issues.

You're mentioning that I can forget about it if I want to promote manually which is exactly what I'd like to do. However, it seems deno deploy instantly promotes the latest deploy to production. This is the behaviour I'd like to turn off.
Enabling me to deploy, but to manually promote a deploy.

From your last sentence I got the feeling this is possible, but how?
